tokio-util/CHANGELOG.md

```diff
@@ -1,3 +1,27 @@
+# 0.7.12 (September 5th, 2024)
+
+This release bumps the MSRV to 1.70. ([#6645])
+
+### Added
+- sync: Add `run_until_cancelled` to `tokio_util::sync::CancellationToken` ([#6618])
+- task: add `AbortOnDropHandle` type ([#6786])
+
+### Changed
+- deps: no default features for hashbrown ([#6541])
+- time: wake `DelayQueue` when removing last item ([#6752])
+- deps: enable the full feature when compiled for the playground ([#6818])
+
+### Documented
+- task: fix typo in `TaskTracer` docs ([#6792])
```

Comment:
There's a typo in the typo fix, how ironic.
```suggestion
- task: fix typo in `TaskTracker` docs ([#6792])
```

Reply:
Indeed. Fixed it.